google/cloud/pubsub_v1/subscriber/_protocol/streaming_pull_manager.py

```diff
@@ -1007,22 +1020,84 @@ def _get_initial_request(
         return request
 
     def _send_lease_modacks(
-        self, ack_ids: Iterable[str], ack_deadline: float, warn_on_invalid=True
+        self,
+        ack_ids: Iterable[str],
+        ack_deadline: float,
+        opentelemetry_data: List[SubscribeOpenTelemetry],
+        warn_on_invalid=True,
+        receipt_modack: bool = False,
     ) -> Set[str]:
         exactly_once_enabled = False
+
+        modack_span: Optional[trace.Span] = None
+        if self._client.open_telemetry_enabled:
+            subscribe_span_links: List[trace.Link] = []
+            subscribe_spans: List[trace.Span] = []
+            assert len(self._subscription.split("/")) == 4
```

Comment:
nit: split this just once and save the result

Reply:
Done.